test/whl-scale-tests/common.ps1

```diff
@@ -0,0 +1,68 @@
+$ErrorActionPreference = 'Stop'
+
+function Get-RandomPassword {
+    param (
+        [Parameter(Mandatory)]
+        [ValidateRange(4,[int]::MaxValue)]

```

Comment:
Since you're adding a validation here, be aware that a length argument that's actually `[int]::MaxValue` here can exceed supported array dimensions which will cause `$bytes = New-Object byte[]($length)` to fail and the function will end up in an infinite recursive loop since `$valid` will not be set. Also I think the limit for password where you use this one is like 256 char or something.

Reply:
Nice catch. I found that the max the VM will expect is 72 characters, so I am changing it to that.